lib/tapioca/generator.rb

```diff
@@ -44,6 +44,38 @@ def build_gem_rbis(gem_names)
       say("Please review changes and commit them.", [:green, :bold])
     end
 
+    sig { void }
+    def build_requires
+      requires_path = Config::DEFAULT_POSTREQUIRE
+      compiler = Compilers::RequiresCompiler.new(Config::SORBET_CONFIG)
+      name = set_color(requires_path, :yellow, :bold)
+      say("Compiling #{name}, this may take a few seconds... ")
+
+      rb_string = compiler.compile
+      if rb_string.empty?
+        say("Nothing to do", :green)
+        return
+      end
+
+      # Clean all existing requires before regenerating the list so we update
+      # it with the new one found in the client code and remove the old ones.
+      File.delete(requires_path) if File.exist?(requires_path)
+
+      content = String.new
+      content << rbi_header(config.generate_command, "false")
```

Comment:
Ugh, this method name sticks out now that we are using it in a non-RBI setting. We need to change that at some point to something like `generated_file_header`. Feel free to do it as part of this PR.